### PR TITLE
Update blob trigger template

### PR DESCRIPTION
**Pull request by aishwaryabh**

### CODE REVIEW

File: Functions.Templates/Templates/BlobTrigger-CSharp-Isolated/.template.config/template.json [hunk 1]
```diff
@@ -51,7 +51,7 @@
             "ManualInstructions": [],
             "args": {
                 "referenceType": "package",
-            "reference": "Microsoft.Azure.Functions.Worker.Extensions.Storage", "version": "4.0.4",
+            "reference": "Microsoft.Azure.Functions.Worker.Extensions.Storage", "version": "5.1.1-preview2",
```
Comment (kshyju) [inline Functions.Templates/Templates/BlobTrigger-CSharp-Isolated/.template.config/template.json:54]: Do we usually include _preview_ version of packages in our templates? I was under the impression that we do not.

Comment (aishwaryabh) [reply Functions.Templates/Templates/BlobTrigger-CSharp-Isolated/.template.config/template.json:54]: We wanted to update the template to reflect the sdk type binding changes and that's in preview today.

**Comment by aishwaryabh** [comment]
Replacing this PR with another one due to weird dependency issues

File: Functions.Templates/Templates/BlobTrigger-CSharp-Isolated/.template.config/template.json (hunk 1 above)
Comment (liliankasem) [reply Functions.Templates/Templates/BlobTrigger-CSharp-Isolated/.template.config/template.json:54]: We don't ship templates with preview versions, this change will ultimately be for the GA package. So we should test the templates with the preview package but update to the stable (GA) package before merging

**Pull request closed without merging**